### DISCUSSION TITLE
Write data concurrency issue

### DISCUSSION THREAD

**Discussion by liekkas99** [OpenAMP/open-amp#517] [Q&A]
I just took over a project based on Openamp, and my colleague told me that there is a bug on the 5.0 version system. When the upper or lower computer writes data to multiple channels, the underlying access is to a file. The system did not process this file bag properly, resulting in the upper or lower computer not receiving data. Has this bug been fixed? In which version was it fixed? How do I obtain this version? Looking forward to your reply, thank you

**Comment by arnopo** [comment]
Hello @liekkas99 
Your request is very unclear. If you need support please first clarify and detail your issue ( environment, usecase, exact race condition...)
I convert this issue to discussion
